inc/ticket.class.php

```diff
@@ -1838,7 +1838,6 @@ function prepareInputForAdd($input) {
                if ($item = getItemForItemtype($itemtype)) {
                   $item->getFromDB($items_id);
                   $input['items_locations'] = $item->fields['locations_id'];
-                  $input['items_groups'] = $item->fields['groups_id'];
```

Comment:
I do not understand why you remove it completely. Why not to solve as it has been in https://github.com/glpi-project/glpi/pull/1471/files?

Reply:
The same is done after line 1847:
```
if (isset($item->fields['groups_id'])) {
    $input['items_groups'] = $item->fields['groups_id'];
 }
```

Reply:
Indeed, you're right